### PR TITLE
drivers: npmx: npmx_shell: Add vbusin status cc get command

### PR DESCRIPTION
**Pull request by magp-nordic**

### CODE REVIEW

File: drivers/npmx/npmx_shell.c [hunk 1]
```diff
@@ -1559,6 +1559,36 @@ static int cmd_vbusin_status_connected_get(const struct shell *shell, size_t arg
...
+	}
+
+	npmx_vbusin_cc_t cc1;
+	npmx_vbusin_cc_t cc2;
+	npmx_vbusin_t *vbusin_instance = npmx_vbusin_get(npmx_instance, 0);
+	npmx_error_t err_code = npmx_vbusin_cc_status_get(vbusin_instance, &cc1, &cc2);
+
+	if (check_error_code(shell, err_code)) {
+		if (cc1 == NPMX_VBUSIN_CC_NOT_CONNECTED) {
+			shell_print(shell, "Value: %d.", cc2);
```
Comment (wilu-nordic) [inline drivers/npmx/npmx_shell.c:1581]: String message is better here: 
`shell_print(shell, "Value: %s.", npmx_vbusin_cc_status_map_to_string(cc2));`

Comment (magp-nordic) [reply drivers/npmx/npmx_shell.c:1581]: Yes, I agree, but I am not sure if people working on PowerUp would not prefer the returned value to be an integer. @masz-nordic what do you think?

**Comment by magp-nordic** [comment]
CI fails due to changed `vbus status get` command structure

File: drivers/npmx/npmx_shell.c (hunk 1 above)
Comment (masz-nordic) [reply drivers/npmx/npmx_shell.c:2728]: Integer is preferred.

**Review by masz-nordic — APPROVED** [review]

**Comment by masz-nordic** [comment]
> CI fails due to changed `vbus status get` command structure

@nordic-segl can you align CI to this change?

**Comment by nordic-segl** [comment]
> > CI fails due to changed `vbus status get` command structure
> 
> @nordic-segl can you align CI to this change?

Change to CI:
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-sdk-npmx/pull-requests/156/overview
waiting for results from both PR-156 in test repo and this PR
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_develop_samples/view/change-requests/job/PR-156/
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_integration/view/change-requests/job/PR-156/

**Review by masz-nordic — APPROVED** [review]

**Review by wilu-nordic — APPROVED** [review]

**Comment by nordic-segl** [comment]
Integration CI is passing on a PR to test repository:
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_integration/view/change-requests/job/PR-156/

**Pull request merged**